cadc-vosi/src/main/java/ca/nrc/cadc/vosi/AvailabilityServlet.java

```diff
@@ -187,19 +207,65 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
         }
     }
 
-    private class ChangeServiceState implements PrivilegedExceptionAction {
-        private AvailabilityPlugin ws;
-        private HttpServletRequest req;
+    /**
```

Comment:
This looks awfully similar to the code in `logControl`. Can it be factored out somewhere at least the list of authorized DNs and group URIs. Maybe a method in `PropertiesReader` or a subclass?

Reply:
The two pieces of code are very similar. Wouldn't be a useful util method to add to the util package however. Had a look through the code and couldn't find another use case for specifying a type to return when get values from the PropertiesReader. If another use case does arise can revisit updating the PropertiesReader.